Pages/Button/button.css

```diff
@@ -0,0 +1,66 @@
+.btn{
+    border: none;
+    padding: 0.5rem 1rem;
+    text-align: center;
+    background-color: var(--primary-color);
+    border-radius: 0.3rem;
+    font-size: 1rem;
+    font-weight: 500;
+    cursor: pointer;
+}
+
+.btn-primary{
+    background-color: var(--primary-color);
+    color: white;
+}
+
+.btn-secondary{
+    background-color: var(--secondary-color);
+    border: 1px solid var(--primary-color);
+}
+
+.btn-success{
+    background-color: var(--success-color);
+}
+
+.btn-warning{
+    background-color: var(--warning-color);
+}
+
+.btn-danger{
+    background-color: var(--danger-color);
+}
+
+.btn-link{
+    text-decoration: underline;
+}
+
+.btn-link > a {
+    text-decoration: none;
+}
+
+.btn-icon{
+    display: inline-flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.btn-icon > img {
+    width: 1.25rem;
+    vertical-align: middle;
+    margin-right: 0.4rem;
+}
+
+.btn-float{
+    border-radius: 50%;
+    min-height: 3.125rem;
+    min-width: 3.125rem;
+    position: fixed;
+}
+
+.btn-float-demo{
+    border-radius: 50%;
+    min-height: 3.125rem;
+    min-width: 3.125rem;
+    /* position: fixed; */
```

Comment:
Just a suggestion, could you remove unwanted comments.
Unless it serves a purpose. Then ignore this comment.

Reply:
Thanks for the suggestion! Will do.. Just kept that line there to show that the floating action should use fixed positioning but didnt use it there since it was a demo in the docs